api/apps/geoprocessing/src/modules/planning-area/marxan-geo-json.ts

```diff
@@ -0,0 +1,32 @@
+import { BBox, GeoJSON } from 'geojson';
+import { PuSizes } from './pu-sizes';
+
+export class MarxanGeoJson {
+  private readonly geoJson: GeoJSON & {
+    marxanMetadata?: Record<string, unknown>;
+  };
+
+  constructor(geoJson: GeoJSON) {
+    this.geoJson = geoJson;
+  }
+
+  addBBox(bbox: BBox) {
+    this.geoJson.bbox = bbox;
+  }
+
+  addPuSizes(puSizes: PuSizes) {
+    this.addMarxanMetadata('maxPuAreaSize', puSizes.maxPuAreaSize);
+    this.addMarxanMetadata('minPuAreaSize', puSizes.minPuAreaSize);
+  }
+
+  addMarxanMetadata<V>(key: string, value: V) {
+    this.geoJson.marxanMetadata = {
```

Comment:
shouldn't this land in `properties` of feature?

Reply:
no, we wanted to have it as a foreign member https://datatracker.ietf.org/doc/html/rfc7946#section-6.1